game_class/item.rb

```diff
@@ -0,0 +1,27 @@
+class Item
```

Comment:
you created a new method in the item class. because this is the main class and every other class inherits from it. you are not going to need this method in all other classes. kindly remove this method from this class and add it to app class.